gradle.properties

```diff
@@ -2,3 +2,4 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.console=verbose
 org.gradle.warning.mode=all
+org.gradle.configuration-cache=true
```

Comment:
This will ensure that gradle configs are cached so we don't spend unnecessary time here